test/functional/feature_governance_cl.py

```diff
@@ -99,16 +98,24 @@ def run_test(self):
 
         assert_equal(len(self.nodes[0].gobject("list", "valid", "triggers")), 0)
 
-        self.log.info("Move 1 block into sb maturity window")
+        self.log.info("Move into sb maturity window")
         n = sb_immaturity_window - self.nodes[0].getblockcount() % sb_cycle
         assert n >= 0
-        for _ in range(n + 1):
+        for _ in range(n):
```

Comment:
consider a follow-up ec65ddeb2dc1471360d26e0100928603d4270276 to improve performance of functional test: from 35 seconds to just 31 (in average of several runs)